lib/engine/game/g_1860.rb

```diff
@@ -124,6 +138,24 @@ def or_set_finished
         check_new_layer
       end
 
+      def place_home_track(corporation)
```

Comment:
why is this method in game? why not just do it in the step, seems like the only place where it is called

Reply:
I was just mirroring place_home_token. I can move it.